### PR TITLE
fix(@angular/cli): fixing lint documentation

### PR DESCRIPTION
**Pull request by sumitarora**
Fixes: 
https://github.com/angular/angular-cli/issues/5632
https://github.com/angular/angular-cli/issues/5713

### CODE REVIEW

**Comment by filipesilva** [comment]
LGTM but pinging @Brocco to review it since he did the original work.

**Comment by sumitarora** [comment]
@johnpapa @coryrylan ☝️

File: docs/documentation/e2e.md [hunk 1]
```diff
@@ -37,17 +37,28 @@ End-to-end tests are run via [Protractor] (https://angular.github.io/protractor/
...
-    `--serve` (alias: `-s`) _default value: true_
+    <code>--serve</code> (aliases: <code>-s</code>) <em>default value: true</em>
   </p>
   <p>
     Compile and Serve the app. All serve options are also available. The live-reload option defaults to false, and the default port will be random.
   </p>
 </details>
 
 <details>
+  <summary>no-serve</summary>
```
Comment (clydin) [inline docs/documentation/e2e.md]: The 'no-' prefix can be used with any Boolean option.  A general statement on each command page would probably be more appropriate than singling out the serve option.

Comment (filipesilva) [inline docs/documentation/e2e.md]: Good catch. @sumitarora can you remove `--no-serve` from this list?

**Comment by hansl** [comment]
Seems like the last 2 commits result in not having any code at all...?

**Comment by sumitarora** [comment]
@hansl I added code and then removed it fixed the commits now

**Pull request merged**